addons/crm/models/crm_lead.py

```diff
@@ -2147,11 +2149,27 @@ def _pls_get_naive_bayes_probabilities(self, batch_mode=False):
         To avoid this, we add 0.1 in each frequency. With few data, the computation is than not really realistic.
         The more we have records to analyse, the more the estimation will be precise.
         :return: probability in percent (and integer rounded) that the lead will be won at the current stage.
+
+        If is_tooltip is True, method instead recomputes the probability of self, that should be a singleton, and instead
```

Comment:
This shows there is some split to do if possible, when you say "with my flag, we actually do something else" :) .